tests/suite/resources_utils.py

```diff
@@ -262,13 +260,9 @@ def wait_until_all_pods_are_ready(v1: CoreV1Api, namespace) -> None:
     :return:
     """
     print("Start waiting for all pods in a namespace to be ContainersReady")
-    counter = 0
-    while not are_all_pods_in_ready_state(v1, namespace) and counter < 50:
-        print("There are pods that are not ContainerReady. Wait for 4 sec...")
-        time.sleep(4)
-        counter = counter + 1
-    if counter >= 50:
-        raise PodNotReadyException()
+    while not are_all_pods_in_ready_state(v1, namespace):
```

Comment:
keeping the logic for 50 or so retries will/can cause these tests to fail esp. with KIC <=1.12.0, but I can probably add condition based on the `BATCH_START`, i.e. if its enabled then logic for giving up after 50 retries should be ignored. Is this okay? @pleshakov

Reply:
@vepatel 
for the condition logical, instead of using the `BATCH_START`, would it make sense to have the max retries as a function parameter. Something like this:
```
def scale_deployment(v1: CoreV1Api, apps_v1_api: AppsV1Api, name, namespace, value, max_retries=50) -> int:
```
or
```
def scale_deployment(v1: CoreV1Api, apps_v1_api: AppsV1Api, name, namespace, value, readiness_timeout=60) -> int:
# 60 seconds
```
In the start up related tests, we can simply increase it to a different bigger value, so that the tests don't fail. what do you think?

Reply:
But we don't have an exact number depending on the number of resources and machine capacity, It'll then become a trial and error case until you find a limit which can accommodate reloads with for eg. say 1000 VSes.

Reply:
I see. well, because `wait_until_all_pods_are_ready` is used in a lot of other tests, at least keeping the retry logic in those makes sense to me.

Reply:
Sure so I'll leave the logic in place and add a comment that disable it if running these tests